public/css/common.less

```diff
@@ -139,6 +139,14 @@ div.show-more {
     padding-top: .25em;
   }
 
+  .control-button {
+    margin-left: .5em;
+
+    &:last-child {
```

Comment:
Please target the `.continue-with` here only. This then would also save us the `margin: 0;` for the sort control.

Besides, another reason why I don't like this is that `:last-child` is not strictly correct here. This only describes the last child in whatever container the control button is a member of. This works currently because the `.filter` container uses the entire width of the column. If it (or any other container with a control button) doesn't require the entire width, this may lead to unexpected results. (As shown with the sort control) By targeting the continue with widget only, we achieve the same result and make it less ambiguous.

Reply:
✅